pandas/core/internals/blocks.py

```diff
@@ -124,7 +124,7 @@ def _simple_new(
         obj._mgr_locs = placement
         return obj
 
-    def __init__(self, values, placement, ndim=None):
+    def __init__(self, values, placement, ndim: int):
```

Comment:
can you add a doc-string here (ok to just doc ndim for for now, but e.g. its 1 for SingleBlockManager and 2 for BlcokManager, always)

Reply:
sure

Reply:
of course, any day now we'll have 2D EAs and ndim wont be necessary at all

Reply:
updated+green